gmso/core/angle.py

```diff
@@ -73,6 +87,24 @@ def restraint(self):
         """Return the restraint of this angle."""
         return self.__dict__.get("restraint_")
 
+    @property
+    def bonds(self):
+        """Return the bond_order symbol of this bond."""
```

Comment:
Update the doc string to say that it returns a tuple of `gmso.core.Bond` instances that make up this angle.